front/app/js/controllers/users/publications/list.html

```diff
@@ -1,15 +1,17 @@
 <div ng-controller="Publications_List_Ctrl" ng-inject="{events: events, newsItems: newsItems}">
     <div infinite-scroll="next()" infinite-scroll-distance="2" infinite-scroll-disabled="$root.infiniteLoading">
 
-        <article col class="Ticket  Ticket--color{{ publication.slug.length%5+1 }}"
+        <article col class="Ticket Ticket--{{ pub_info[publication.publication_state].color }}"
             ng-repeat="publication in events.data | concat: newsItems.data | orderBy: 'date' | reverse"
-            ng-if="publication.entry_method != 'Ferie'">
+            ng-if="publication.entry_method != 'Ferie' && ($root.hasClub(publication.author_club.slug) || ['Published', 'Emailed'].includes(publication.publication_state))">
+            <up-ribbon ng-repeat="(state, item) in pub_info" class="Ribbon--{{ pub_info[publication.publication_state].color }}" ng-hide="editSlug == publication.slug"
```

Comment:
pourquoi tu boucles alors que tu sais que `publication.publication_state == state` ?

Reply:
Il y a 4 ribbons différents, un membre voit les publications de son club pour les 4 états.
On est dans list.html à l'intérieur d'un `ng-repeat="publication in events.data | concat: newsItems.data | orderBy: 'date' | reverse"` donc publication.publication_state change dans ce feed de publications